container/docker/handler.go

```diff
@@ -160,8 +163,12 @@ func newDockerContainerHandler(
 		fsInfo:             fsInfo,
 		rootFs:             rootFs,
 		rootfsStorageDir:   rootfsStorageDir,
-		fsHandler:          newFsHandler(time.Minute, rootfsStorageDir, otherStorageDir, fsInfo),
 		envs:               make(map[string]string),
+		ignoreMetrics:      ignoreMetrics,
+	}
+
+	if _, ignore := ignoreMetrics[container.DiskUsageMetrics]; !ignore {
```

Comment:
(nit) If you make ignoreMetrics a `map[MetricKind]bool`, then these checks become cleaner. E.g.
```go
if !ignoreMetrics[container.DiskUsageMetrics] {
```